### ISSUE TITLE
Responsive Box Hover Effect

### ISSUE THREAD

**Issue by shallanidevi** [shallanidevi/CSS-mini-projetcs#1]
i will add the code for Responsive Box Hover Effect

**Comment by jeel2410** [comment]
@shallanidevi can you give me this work? I definitely completed this work

**Comment by shallanidevi** [comment]
@jeel2410 Yes definity but don't copy-paste someone else  code. Try writing your own code.
